scripts/deploy-gh-pages.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+
+mkdir tmp
+cd tmp
+
+SOURCE_BRANCH="master"
+TARGET_BRANCH="gh-pages"
+
+# Pull requests and commits to other branches shouldn't try to deploy, just build to verify
+if [ "$TRAVIS_PULL_REQUEST" != "false" -o -z "$TRAVIS_TAG" ]; then
```

Comment:
The list of variables is [here](https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables) and you can see "`TRAVIS_TAG: If the current build is for a git tag, this variable is set to the tag’s name.`" So, I found that I can remove this compound because I specify the trigger [here](https://github.com/PileProject/drivecommand/pull/22/files#diff-354f30a63fb0907d4ad57269548329e3R24) and that requires a tag (and thus, it cannot be a PR).